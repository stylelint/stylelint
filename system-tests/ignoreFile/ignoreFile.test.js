'use strict';

const stylelint = require('../../lib');
const systemTestUtils = require('../systemTestUtils');

it('ignoreFile', () => {
	return stylelint
		.lint({
			files: [systemTestUtils.caseStylesheetGlob('ignoreFile')],
			ignorePath: require('path').join(__dirname, '.stylelintignore'),
			allowEmptyInput: true,
			configFile: systemTestUtils.caseConfig('ignoreFile'),
		})
		.then((output) => {
			expect(output).toMatchSnapshot();
		});
}, 10000);
