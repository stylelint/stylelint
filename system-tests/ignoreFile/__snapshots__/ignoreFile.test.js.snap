// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ignoreFile 1`] = `
Object {
  "errored": false,
  "output": "[]",
  "results": Array [],
}
`;
